SPECS/procps-ng/procps-ng.spec

```diff
@@ -1,35 +1,40 @@
 Summary:        Programs for monitoring processes
 Name:           procps-ng
-Version:        3.3.15
-Release:        5%{?dist}
-License:        GPLv2
-URL:            http://procps.sourceforge.net/
-Group:          Applications/System
+Version:        3.3.17
+Release:        1%{?dist}
+License:        GPLv2 AND LGPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Applications/System
+URL:            http://procps.sourceforge.net/
```

Comment:
can we use https - this redirects to - https://gitlab.com/procps-ng/procps

Reply:
Fixed to point to gitlab URL